guides/v2.2/cloud/reference/cloud-composer.md

```diff
@@ -45,50 +37,27 @@ The workflow is as follows:
-* `composer update`: Use this command when you need to perform complete updates of all extensions and base code. This command uses `composer.json`, not `composer.lock`.
-
-## Magento Commerce (Cloud) packages
-The following sections discuss the Composer packages used by {{site.data.var.ece}}:
-
-*	[`vendor/magento/magento-cloud-metapackage`](#cloud-composer-cloudmeta)
-*	[`vendor/magento/ece-patches`](#ece-patches)
-*	[`vendor/magento/ece-tools`](#ece-tools)
-*	[`vendor/magento/product-enterprise-edition`](#cloud-composer-prodee)
-
-<div class="bs-callout bs-callout-info" id="info" markdown="1">
-Refer to [Composer package updates]({{ site.baseurl }}/guides/v2.2/cloud/composer-packages/patch-notes.html) for release information.
-</div>
-
-### magento/magento-cloud-metapackage {#cloud-composer-cloudmeta}
-`vendor/magento/magento-cloud-metapackage` should be the only package in the `require` section of your `composer.json`. This is a [_metapackage_](https://getcomposer.org/doc/04-schema.md#type){:target="\_blank"} and does not contain any code.
+## magento/magento-cloud-metapackage {#cloud-composer-cloudmeta}
+The `vendor/magento/magento-cloud-metapackage` should be the only package in the `require` section of your `composer.json`. This is a [_metapackage_](https://getcomposer.org/doc/04-schema.md#type){:target="\_blank"} and does not contain any code.
 
 The metapackage depends on the appropriate versions of [`vendor/magento/ece-patches`](#ece-patches), [`vendor/magento/ece-tools`](#ece-tools), and [`vendor/magento/product-enterprise-edition`](#cloud-composer-prodee). At any given version, this package requires the same version of `magento/product-enterprise-edition`. For example, to use {{site.data.var.ee}} version 2.2.0, for example, `composer.json` must specify a requirement for `magento/magento-cloud-metapackage` version 2.2.0.
 
 This package depends on a floating version of `vendor/magento/magento-cloud-configuration` (abbreviated _MCC_). It depends on the major and minor version of MCC that correspond to the specified {{site.data.var.ee}} version, and floats on the patch version so that compatible updates to this packages can be automatically pulled by running `composer update`.
 
-### vendor/magento/ece-patches {#ece-patches}
-
-{% include cloud/ece-patches.md %}
-
-### vendor/magento/ece-tools {#ece-tools}
-
-{% include cloud/ece-tools.md %}
+## vendor/magento/ece-tools {#ece-tools}
+The `ece-tools` package, also known as _ece-tools_, is compatible with Magento version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ee}} project.  It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your Cloud projects.
```

Comment:
I think we need to be more precise in how we're referring to products in this paragraph.

- Use the Commerce variable to refer to Magento 2.1.4
- Use the Magento Commerce (Cloud) variable to refer to managing your projects
- Drop the "Cloud" reference at the end of the sentence

> "The `ece-tools` package, also known as _ece-tools_, is compatible with ~Magento~ {{site.data.var.ee}} version 2.1.4 and later to provide a rich set of features you can use to manage your ~{{site.data.var.ee}}~{{site.data.var.ece}} project.  It contains scripts and commands designed to help manage your code and automatically build and deploy your projects.